docs/built-in-models/linear.md

```diff
@@ -28,6 +28,9 @@ The `n_bits` parameter controls the bit-width of the inputs and weights of the l
 
 The `n_bits` parameter can be set to `8` or more bits for models with up to `300` input dimensions. When the input has more dimensions, `n_bits` must be reduced to `6-7`. All performance metrics are preserved down to `n_bits=6`, compared to the non-quantized float models from scikit-learn.
 
+The same quantization parameters (i.e., scale and zero-point) are applied on all features, so it can be beneficial to uniformize the distribution of the different features.
```

Comment:
Not sure about the 
```
The same quantization parameters (i.e., scale and zero-point) are applied on all features, so it can be beneficial to uniformize the distribution of the different features.
```

Maybe something like:
```suggestion
The same quantization parameters (i.e., scale and zero-point) are applied on all features, so it can be beneficial to make all feature distribution similar by using standard or min-max normalization.
```